integrations/nielsen-dcr/lib/index.js

```diff
@@ -164,53 +153,57 @@ NielsenDCR.prototype.heartbeat = function(assetId, position, options) {
  */
 
 NielsenDCR.prototype.getContentMetadata = function(track, type) {
-  var properties = 'properties.';
-  var assetIdProp = 'asset_id';
-  if (type) {
-    if (type === 'preroll') properties = 'properties.content.';
-    if (type === 'video') assetIdProp = 'content_asset_id';
```

Comment:
@brennan Should we still be differentiating between video and ad in this method?